.github/actions/amplify-preview/action.yaml

```diff
@@ -0,0 +1,141 @@
+name: Amplify Preview
+description: Prepare Amplify Preview URL and post it in PR comments
+inputs:
+  app_ids:
+    description: "Comma separated list of Amplify App IDs"
+    required: true
```

Comment:
This is actually meaningless - GHA doesn't check it for composite actions. A step in the action needs to validate it instead